interfaces/IBF-dashboard/src/app/config.ts

```diff
@@ -43,3 +43,10 @@ export const LEAFLET_MAP_OPTIONS: MapOptions = {
   zoom: 5,
   layers: [],
 };
+
+export const DISASTER_TYPES_SVG_MAP = {
+  'floods': '/assets/icons/Flood.svg',
+  'dengue': '/assets/icons/Mosquito.svg',
+  'malaria': '/assets/icons/Mosquito.svg',
+  'heavy rainfal': '/assets/icons/Alert.svg',
```

Comment:
Please use the [Disaster Type Key](https://github.com/rodekruis/IBF-system/blob/master/interfaces/IBF-dashboard/src/app/types/disaster-type-key.ts#L1) instead of the Disaster Type Label.

This will help keep consistent and also avoid typos as in [line 51](https://github.com/rodekruis/IBF-system/pull/810/files#diff-f64cdb1ffcfa51a8e45821b7790d549ca8f6f3b2aff5b34164710ef45ccea56eR51).

Reply:
Taken care